client/app/private-database/database/add/private-database-database-add.controller.js

```diff
@@ -33,10 +33,10 @@ angular.module("App").controller(
                     value: "",
                     condition: {
                         length: {
-                            min: 3,
+                            min: 1,
```

Comment:
Can we get the information from API ?

Reply:
I had a look and API doesn't give that kind of information unfortunately